### PR TITLE
Add argocd scrape config

### PR DESCRIPTION
**Pull request by HumairAK**
Add external scrape configs (the lack of newline in the configs is intentional)

### CODE REVIEW

**Comment by HumairAK** [comment]
also added ci config files with the pr to avoid build checks

File: monitoring/psi-prod-cluster/overlays/prod/prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  labels:
+    prometheus: aicoe-sre
+  name: aicoe-sre-prometheus
```
Comment (anishasthana) [inline monitoring/psi-prod-cluster/overlays/prod/prometheus.yaml]: I wonder if we should change the name.

Comment (HumairAK) [reply monitoring/psi-prod-cluster/overlays/prod/prometheus.yaml]: that would probably come in a different pr, this one is just a patch on the original base with the same name

Comment (HumairAK) [reply monitoring/psi-prod-cluster/overlays/prod/prometheus.yaml]: updated the name (might as well)

**Review by anishasthana — APPROVED** [review]

**Pull request merged**